pipeline.yml

```diff
@@ -730,16 +730,33 @@ jobs:
         ENCRYPTION: *encryption
 
 resource_types:
+- name: git
+  type: registry-image
+   source:
+     aws_access_key_id: ((ecr_aws_key))
+     aws_secret_access_key: ((ecr_aws_secret))
+     repository: git-resource
+     aws_region: us-gov-west-1
+     semver_constraint: ">= 1.0.0"
```

Comment:
I think instead of `semver_constraint` we want to use `tag: latest`

Reply:
Thank you David. I have made the changes.